driver-legacy/src/main/com/mongodb/DBCursor.java

```diff
@@ -518,7 +518,10 @@ public DBCursor snapshot() {
      * @return a {@code DBObject} containing the explain output for this DBCursor's query
      * @throws MongoException if the operation failed
      * @mongodb.driver.manual reference/explain Explain Output
+     * @deprecated This method is no longer part of our CRUD API. In 4.x this method will only support MongoDB 3.2 and up (where the
```

Comment:
This isn't accurate.  In 4.x we are going to remove the method.  How about

    @deprecated Replace with direct use of the explain command using the runCommand helper method

Reply:
I forgot that we would remove the method in 4.x. This comment is better. I have made the change.